src/services/cacheConfigUtils.ts

```diff
@@ -154,53 +154,54 @@ export class CacheConfigUtils {
 
   public static createCacheConfigFromData(data: CacheConfiguration): string {
     let cache;
+    let cacheType;
+
+    const distributedCache = 'distributed-cache';
+    const replicatedCache = 'replicated-cache';
+
+    const generalCache = {
+      mode: data.basic.mode,
+      owners: data.basic.numberOfOwners,
+      statistics: data.basic.statistics,
+      encoding: {
+        'media-type': data.basic.encoding,
+      },
+      locking: {
+        isolation: data.advanced.isolationLevel,
+        striping: data.advanced.striping,
+        'concurrency-level': data.advanced.concurrencyLevel,
+        'acquire-timeout': data.advanced.lockAcquisitionTimeout,
+      },
+    };
+
+    data.basic.topology === 'Distributed'
+      ? ((cache = { [distributedCache]: generalCache }),
+        (cacheType = distributedCache))
+      : ((cache = { [replicatedCache]: generalCache }),
+        (cacheType = replicatedCache));
+
+    const expiration = () => {
+      cache[cacheType]['expiration'] = {
+        lifespan: data.basic.lifeSpan,
+        'max-idle': data.basic.maxIdle,
+
+        // Todo: Find Proper variable name
```

Comment:
@karesti @oraNod For reaper and wake up interval, what would be the key name? can't find on documentation.

Reply:
@dipguptaredhat Could it be here? https://github.com/infinispan/infinispan/blob/main/core/src/main/java/org/infinispan/configuration/cache/ExpirationConfiguration.java#L18